connectors/src/connectors/zendesk/lib/zendesk_api.ts

```diff
@@ -134,6 +134,13 @@ async function fetchFromZendeskWithRetries({
   try {
     response = await rawResponse.json();
   } catch (e) {
+    if (rawResponse.status === 404) {
```

Comment:
The 404 was not handled by line 156??

Reply:
actually 404 crash at parsing, which will cause it to return an error